openmc/stats/multivariate.py

```diff
@@ -779,3 +779,136 @@ def from_xml_element(cls, elem):
         """
         xyz = [float(x) for x in get_text(elem, 'parameters').split()]
         return cls(xyz)
+
+
+class SphericalShell(Spatial):
```

Comment:
I don't think we actually need a new _class_ for this. Instead, I would just write a plain function that returns an instance of `SphericalIndependent` with the appropriate distributions set. The function could look something like:
```Python
def spherical_uniform(r_outer, r_inner=0.0, thetas=(0., pi), phis=(0., 2*pi), origin=(0., 0., 0.)):
    ...
    return SphericalIndependent(...)
```
With these argument defaults, in the simplest case where the user just wants a uniform spherical source, it would just be `dist = spherical_shell(radius)`. The beauty of this is that we don't need to write any new XML writing- or reading methods; one less piece for us to maintain.

Reply:
yes